iam-login-service/src/main/java/it/infn/mw/iam/api/account/password_reset/ResetPasswordDTO.java

```diff
@@ -0,0 +1,37 @@
+
+/**
+ * Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2021
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package it.infn.mw.iam.api.account.password_reset;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+import org.hibernate.validator.constraints.Length;
+
+public class ResetPasswordDTO {
+  @NotEmpty(message = "The password cannot be empty")
+  @Length(min = 8, message = "The password must be at least 8 characters")
+  @Pattern(regexp="^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$", message = "The password must be strong")
```

Comment:
It's a copy of the other REGEXP. I'd try to have this REGEXP as a constant stored somewhere (Constants, Utils, ..?)